app/controllers/concerns/saml_idp_auth_concern.rb

```diff
@@ -85,8 +86,9 @@ def requested_ial_authn_context
   end
 
   def link_identity_from_session_data
-    IdentityLinker.new(current_user, current_issuer).
-      link_identity(ial: ial_context.ial_for_identity_record)
+    IdentityLinker
+      .new(current_user, current_issuer)
+      .link_identity(ial: ial_context.ial_for_identity_record)
```

Comment:
would be curious about other's thoughts on this:

Personally I like the leading dot instead of trailing dot for multi-line chained method calls

However, in Ruby it's really advantageous to have trailing dot, because then the code can be copy-pasted to the REPL whereas with leading dot, it can't

Reply:
I agree I like the leading dot... I agree we should be able to paste into the repl... ¯\_(ツ)_/¯

Reply:
I also think a shrug should be easier to type than `¯\\\_(ツ)\_/¯ `  (╯ಠ益ಠ)╯彡 ┻━┻